sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -86,9 +86,9 @@
         WHEN -1 THEN DATALENGTH(text)
         ELSE statement_end_offset END
             - statement_start_offset) / 2) + 1) AS statement_text,
-    qt.text,
+    SUBSTRING(qt.text, 1, {proc_char_limit}),
```

Comment:
Do we need to add the alias `text` to `SUBSTRING...` (`SUBSTRING(qt.text, 1, {proc_char_limit} as text`) , so that the truncated text will be correctly picked up by the Python code e.g. [here](https://github.com/DataDog/integrations-core/blob/94a5de4d28ae51bc4064e220c379b4d026704093/sqlserver/datadog_checks/sqlserver/statements.py#L295) ?

Reply:
should the alias text be after the substring? like `SUBSTRING(qt.text, 1, {proc_char_limit}) as text`?

Reply:
Yes - I misplaced the bracket in my comment.